velox/vector/VectorStream.h

```diff
@@ -125,6 +159,10 @@ class VectorSerde {
       StreamArena* streamArena,
       const Options* options = nullptr) = 0;
 
+  virtual std::unique_ptr<BatchVectorSerializer> createBatchSerializer(
```

Comment:
nit: Is it better to call this VectorGroupSerializer? s/createBatchSerializer/createVectorGroupSerializer/